internal/local/structure_v2.go

```diff
@@ -556,29 +648,7 @@ func writeDataSources(rootDir string, dataSources []DataSourceStructure) error {
 			return err
 		}
 		for _, rule := range ds.Rules {
-			schema := rule[NameSchema]
```

Comment:
I think we still want to keep this code path here, if the `rule` happens to have these things, `Write` should account for it